docs/cpp/source/frontend.rst

```diff
@@ -45,55 +45,62 @@ neural network on the MNIST dataset:
   struct Net : torch::nn::Module {
     Net() {
       // Construct and register two Linear submodules.
-      fc1 = register_module("fc1", torch::nn::Linear(8, 64));
-      fc2 = register_module("fc2", torch::nn::Linear(64, 1));
+      fc1 = register_module("fc1", torch::nn::Linear(784, 64));
+      fc2 = register_module("fc2", torch::nn::Linear(64, 32));
+      fc3 = register_module("fc3", torch::nn::Linear(32, 10));
     }
 
     // Implement the Net's algorithm.
     torch::Tensor forward(torch::Tensor x) {
       // Use one of many tensor manipulation functions.
-      x = torch::relu(fc1->forward(x));
-      x = torch::dropout(x, /*p=*/0.5, /*train=*/true);
-      x = torch::sigmoid(fc2->forward(x));
+      x = torch::relu(fc1->forward(x.resize_({x.size(0), 784})));
```

Comment:
`resize_` should ever be used in user code. It's a low-level function.
You are either looking for `.view` or `.reshape`

Reply:
Fixed